asserts/database.go

```diff
@@ -113,15 +114,20 @@ func OpenDatabase(cfg *DatabaseConfig) (*Database, error) {
 		panic("database cannot be used without setting a keypair manager")
 	}
 
-	trustedKeys := make(map[string][]*AccountKey)
+	trustedBackstore := NewMemoryBackstore()
+
 	for _, accKey := range cfg.TrustedKeys {
-		authID := accKey.AccountID()
-		trustedKeys[authID] = append(trustedKeys[authID], accKey)
+		err := trustedBackstore.Put(AccountKeyType, accKey)
+		if err != nil {
+			return nil, fmt.Errorf("error populating trusted account keys with key for %q key id %q: %v", accKey.AccountID(), accKey.PublicKeyID(), err)
```

Comment:
Got a bit lost reading too many "keys" in sequence. How about

`"error loading trusted account key id %q for %q: %v"`

("populating" feels strange for some reason)